ui/nuxeo-video/nuxeo-video-info.js

```diff
@@ -0,0 +1,88 @@
+/**
+@license
+(C) Copyright Nuxeo Corp. (http://nuxeo.com/)
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import { html } from '@polymer/polymer/lib/utils/html-tag.js';
+import { mixinBehaviors } from '@polymer/polymer/lib/legacy/class.js';
+import { I18nBehavior } from '../nuxeo-i18n-behavior';
+
+{
+  /**
+  `nuxeo-video-info`
+  @group Nuxeo UI
+  @element nuxeo-video-info
+  */
+
+  class VideoInfo extends mixinBehaviors([I18nBehavior], Nuxeo.Element) {
+    static get template() {
+      return html`
+        <style>
+          .properties {
+            column-width: 25em;
+            column-count: 3;
+          }
+
+          .properties label {
+            width: 10em;
+            overflow: hidden;
+            text-overflow: ellipsis;
+          }
+
+          .item {
+            @apply --layout-horizontal;
+            @apply --layout-flex;
+          }
+
+          .item label {
+            color: var(--secondary-text-color, #939caa);
+          }
+        </style>
+        <div class="properties">
+          <div class="item">
+            <label>[[i18n('videoViewLayout.format')]]</label>
+            <div>[[document.properties.vid:info.format]]</div>
+          </div>
+          <div class="item">
+            <label>[[i18n('videoViewLayout.duration')]]</label>
+            <div>[[document.properties.vid:info.duration]]</div>
+          </div>
+          <div class="item">
+            <label>[[i18n('videoViewLayout.width')]]</label>
+            <div>[[document.properties.vid:info.width]]</div>
+          </div>
+          <div class="item">
+            <label>[[i18n('videoViewLayout.height')]]</label>
+            <div>[[document.properties.vid:info.height]]</div>
+          </div>
+          <div class="item">
+            <label>[[i18n('videoViewLayout.frameRate')]]</label>
+            <div>[[document.properties.vid:info.frameRate]]</div>
+          </div>
+        </div>
+      `;
+    }
+
+    static get is() {
+      return 'nuxeo-video-info';
+    }
+
+    static get properties() {
+      return {
+        document: Object,
```

Comment:
Documentation is still missing for this property.

Reply:
thanks, just added it